static/styles/settings.css

```diff
@@ -752,7 +752,7 @@ input[type="checkbox"] {
         cursor: move;
 
         .fa-ellipsis-v {
-            color: hsl(0, 0, 75%);
+            color: hsl(0, 0%, 20%);
```

Comment:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/25124304/157256317-944cd3b4-3c93-48fc-9f19-91eca4f74383.png">

Reply:
These are supposed to be grab handles to indicate sortability; lower contrast makes sense. See the equivalent component in [Material](https://material.io/components/lists#behavior) (search for “Items can be dragged to reorder a list”).

It seems this particular bug is already compiled away by production CSS minification, so I’m inclined to leave this at `hsl(0, 0%, 75%)`. Perhaps `hsla(0, 0%, 50%, 50%)` would be an improvement for dark mode consistency.